### PR TITLE
Add Tracing Doc

### PR DESCRIPTION
**Pull request by Alima777**

### CODE REVIEW

File: docs/UserGuide/Operation Manual/SQL Reference.md [hunk 1]
```diff
@@ -953,6 +954,15 @@ This example will delete the first 3 time partitions of storage group root.sg1.
 ```
 The partitionId can be found in data folders or converted using `timestamp / partitionInterval`.
 
+## 性能追踪
```
Comment (HTHou) [inline docs/UserGuide/Operation Manual/SQL Reference.md:957]: It should be in English...

File: docs/UserGuide/System Tools/Performance Tracing Tool.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+# Performance Tracing Tool
+
+Iotdb supports the use of `TRACING` statements to enable and disable performance tracing of query statements, which is disabled by default. Users can use performance tracking tool to analyze potential performance problems in some queries. By default, the log files for performance tracing are stored in the directory `./data/tracing`.
```
Comment (HTHou) [inline docs/UserGuide/System Tools/Performance Tracing Tool.md:23]:
```suggestion
IoTDB supports the use of `TRACING` statements to enable and disable performance tracing of query statements, which is disabled by default. Users can use performance tracking tool to analyze potential performance problems in some queries. By default, the log files for performance tracing are stored in the directory `./data/tracing`.
```

File: docs/UserGuide/Operation Manual/SQL Reference.md (hunk 1 above)
Comment (Alima777) [reply docs/UserGuide/Operation Manual/SQL Reference.md]: Haha, fixed. :)

File: docs/UserGuide/System Tools/Performance Tracing Tool.md (hunk 2 above)
Comment (Alima777) [reply docs/UserGuide/System Tools/Performance Tracing Tool.md]: Fixed.

**Pull request merged**